supplier_product/tests.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from supplier_product.models import SupplierProduct
+
+
+class TestSupProduct:
+
+    @pytest.mark.django_db()
```

Comment:
If you are using the `@pytest.mark.django_db` mark for all functions in your class, you can apply this mark at the class level. That way you only declare it once and it applies to all of the functions of this class.